### DISCUSSION TITLE
Please tell me how to solve this error

### DISCUSSION THREAD

**Discussion by Reyororo** [FunkinCrew/Funkin#1896] [Compiling help]
I'm using VS Code and I'm getting this error
![スクリーンショット 2021-12-02 124028](https://user-images.githubusercontent.com/88378158/144353657-6809ec1e-61f1-444e-aa45-4159ae2cf3f6.png)

**Comment by Cheemsandfriends** [comment]
Maybe you have a hxcpp that doesn't support 64 bits, or a component that isn't 64bits idk
